File: src/main/java/src/gestorAplicacion/personas/Persona.java
package gestorAplicacion.personas;

import java.io.Serializable;

public abstract class Persona implements Serializable{
	//Atributos clase abstracta
	public String nombre;
	public long ID;
	public String Email;
	
	public Persona (String nombre, long ID, String Email) {
		this.nombre=nombre;
		this.ID = ID;
		this.Email=Email;
	}
	
	//metodos get y set 
	public String getNombre() {
		return nombre;
	}
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	public long getID() {
		return ID;
	}
	public void setID(long ID) {
		this.ID=ID;
	}
	
	public String getEmail() {
		return Email;
	}
	public void setEmail(String Email) {
		this.Email=Email;
	}
}

File: src_py/personas/Persona.py
from abc import ABC

class Persona(ABC):
    def __init__(self, nombre: str, ID: int, Email: str):
        self.__nombre = nombre
        self.__ID = ID
        self.__Email = Email

    def getNombre(self):
        return self.__nombre

    def setNombre(self, nombre):
        self.__nombre = nombre

    def getID(self) -> int:
        return self.__ID

    def setID(self, ID: int):
        self.__ID = ID

    def getEmail(self) -> str:
        return self.__Email

    def setEmail(self, Email: str):
        self.__Email = Email
